frontend/public/locales/en/search.json

```diff
@@ -2,6 +2,7 @@
     "search": "Search",
     "search.filter.errors.description": "The search service is unavailable.",
     "search.filter.errors.title": "An unexpected error occurred.",
+    "search.filter.info.title": "Please read the comment below.",
```

Comment:
This message is just a placeholder. @dockerymick can you provide an appropriate string for the alert title?

Reply:
I would probably say, "Configuration alert" or "Enablement alert"

I am leaning towards Configuration alert

Reply:
Changing it to configuration alert.